src/test/java/com/wikia/webdriver/testcases/adstests/TestVUAP.java

```diff
@@ -9,18 +10,57 @@
 
 public class TestVUAP extends TemplateNoFirstLoad {
   private static final Dimension DESKTOP_SIZE = new Dimension(1920, 1080);
+  private static final String TOP_LEADERBOARD = "TOP_LEADERBOARD";
+  private static final String BOTTOM_LEADERBOARD = "BOTTOM_LEADERBOARD";
+  private static final String FANDOM_URL = "http://www.wikia.com/fandom";
+  private static final String PROJECT43 = "project43";
+  private static final String VUAP_ARTICLE = "SyntheticTests/VUAP";
+
+  private final String currentBrowserTab() {
```

Comment:
I believe test files should keep the test related logic. Consider placing this method in one of existing page objects